doc/content/en/docs/next/api-c++.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "C++ API"
+linkTitle: "C++ API"
+weight: 102
```

Comment:
I spread the weights out a bit, so that the ones in the root of the `next/` artifacts are all in the range 1XX, and the others are either 0XX, 2XX depending on where we want to see them.